src/core/ext/transport/chttp2/server/chttp2_server.cc

```diff
@@ -1078,11 +1080,12 @@ void grpc_server_add_channel_from_fd(grpc_server* server, int fd,
   grpc_core::ChannelArgs server_args = core_server->channel_args();
   std::string name = absl::StrCat("fd:", fd);
   auto memory_quota =
-      grpc_core::ResourceQuotaFromChannelArgs(server_args)->memory_quota();
-  auto config =
-      grpc_event_engine::experimental::CreateEndpointConfig(server_args);
+      grpc_core::ResourceQuotaFromChannelArgs(server_args.ToC().get())
```

Comment:
Why is this being changed to use C-style channel args instead of C++-style?

Reply:
Replaced call-sites everywhere to use C++ style channel args